src/build_loss_objective.jl

```diff
@@ -49,9 +65,22 @@ function build_loss_objective(prob::DiffEqBase.DEProblem,alg,loss,regularization
   if mpg_autodiff
     gcfg = ForwardDiff.GradientConfig(cost_function, autodiff_prototype, autodiff_chunk)
     g! = (x, out) -> ForwardDiff.gradient!(out, cost_function, x, gcfg)
+  elseif lsa_gradient
+    if typeof(loss) <: L2Loss
+      function g!(x,out)
+        sol_,sens = diffeq_sen_full(prob.f,prob.u0,prob.tspan,x,loss.t;alg=alg)
```

Comment:
why is this using the forward sensitivity analysis?

Reply:
We had a conversation the other day when we decided that we will support this as well as the adjoint sensitivity analysis method just in case user in interested. I don't see why not leave it there (it does work and is tested) incase someone would like to use it for some possible use case?

Reply:
ahh yes, lsa instead of adjlsa. Change the name to flsa for forward.